docker/docker-compose.yml

```diff
@@ -0,0 +1,170 @@
+    # Adds a network alias (used in neo4j.conf when configuring the discovery members)
+    networks:
+      db:
+        aliases:
+          - neo4j-network
+    # The ports that will be accessible from outside the container - HTTP (7474) and Bolt (7687).
+    expose:
+      - 7687
+    # Uncomment the volumes to be mounted to make them accessible from outside the container.
+    volumes:
+      #- ./neo4j.conf:/conf/neo4j.conf # This is the main configuration file.
+      - ${NEO4J_DATA_STORE:?err}:/var/lib/neo4j/data
+      #- ./data/server:/var/lib/neo4j/data
+      #- ./logs/neo4j:/var/lib/neo4j/logs
+      #- ./conf/neo4j:/var/lib/neo4j/conf
+      #- ./import/neo4j:/var/lib/neo4j/import
+      #- ./metrics/neo4j:/var/lib/neo4j/metrics
+      #- ./licenses/neo4j:/var/lib/neo4j/licenses
+      #- ./ssl/neo4j:/var/lib/neo4j/ssl
+    # Passes the following environment variables to the container
+    environment:
+      - NEO4J_AUTH=${NEO4J_USER:?err}/${NEO4J_PASS:?err}
+      - NEO4J_EDITION=docker_compose
+    # Simple check testing whether the port 7474 is opened.
+    # If so, the instance running inside the container is considered as "healthy".
+    # This status can be checked using the "docker ps" command.
+    healthcheck:
+      test: ["CMD-SHELL", "wget --no-verbose --tries=1 --spider localhost:7474 || exit 1"]
+    # Set up the user
+    user: ${USER_ID:?err}:${GROUP_ID:?err}
+    labels:
+      - "traefik.enable=false"
+
+  fah-alchemy-client-API:
+    image: mmh42/fah-testing:latest
+    hostname: fahalchemy
+    networks:
+      - internal
+      - db
+    expose:
+      - 1840
+    environment:
+      - JWT_SECRET_KEY=${JWT_SECRET_KEY:?err}
+      - JWT_EXPIRE_SECONDS=${JWT_EXPIRE_SECONDS:-1800}
+      - JWT_ALGORITHM=${JWT_ALGORITHM:-HS256}
+      - AWS_ACCESS_KEY_ID=${AWS_ACCESS_KEY_ID}
+      - AWS_SECRET_ACCESS_KEY=${AWS_SECRET_ACCESS_KEY}
+      - AWS_SESSION_TOKEN=${AWS_SESSION_TOKEN}
+      - AWS_S3_BUCKET=${AWS_S3_BUCKET:?err}
+      - AWS_S3_PREFIX=${AWS_S3_PREFIX:?err}
+      - AWS_DEFAULT_REGION=${AWS_DEFAULT_REGION:?err}
+      - NEO4J_URL=${NEO4J_URL:?err}
+      - NEO4J_USER=${NEO4J_USER:?err}
+      - NEO4J_PASS=${NEO4J_PASS:?err}
+    depends_on: 
+      - neo4j
+      - fah-alchemy-compute-API
+      - fah-alchemy-db-init
+    command: "api --host 0.0.0.0 --port 1840 --workers 2"
+    labels:
+      - "traefik.enable=true"
+      - "traefik.http.routers.fah-alchemy-client-API.rule=Host(`fahalchemy.${HOST_DOMAIN:?err}`)"
+      - "traefik.http.routers.fah-alchemy-client-API.entrypoints=websecure"
+      - "traefik.http.routers.fah-alchemy-client-API.tls.certresolver=myresolver"
+      - "traefik.docker.network=docker_internal"
+    healthcheck:
+      test: ["CMD", "/opt/conda/bin/curl", "-f", "http://localhost:1840/ping"]
+      interval: 1m
+      timeout: 10s
+      retries: 3
+      start_period: 20s
+
+  fah-alchemy-compute-API:
+    image: mmh42/fah-testing:latest
+    networks:
```

Comment:
Does this one also need a `hostname: fahalchemy-compute`?

Reply:
Good catch, it doesn't *need* it, I will actually need to double checks the docs to see if this will affect scaling (as in, will it append something like `-1`, `-2`, as we scale.